### PR TITLE
Bumped version and updateds CHANGELOG

### PR DESCRIPTION
**Pull request by qduk**
Bump version and update CHANGELOG for 1.5 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## v1.5.0 - 2023-01-13
+
+### Added
+
+- #112 - Added tests.
```
Comment (jdrew82) [inline CHANGELOG.md:7]: There was also the option to create a CloudVision device and create RelationshipAssociations to the managed devices in CVP which was #121. Also to track the version for CloudVision in DLC.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+
+### Fixed
+
+- #111 - Ensured site/role mapping to be plural
+- #112 - Removed default status info from README.
+- #118 - Fixed Device Lifecycle plugin integration and AP handling. Addresses #116 and #117.
+- #122 - Fixed hostname parsing. Addresses #119.
```
Comment (jdrew82) [inline CHANGELOG.md:14]: There was a fix for getting information about interfaces that was part of 117.

**Review by jdrew82 — CHANGES_REQUESTED** [review]
Just a few tweaks to the changelog and should be g2g.

File: CHANGELOG.md (hunk 1 above)
Comment (qduk) [reply CHANGELOG.md:7]: Updated.

File: CHANGELOG.md (hunk 2 above)
Comment (qduk) [reply CHANGELOG.md:16]: Updated

**Review by jdrew82 — APPROVED** [review]
LGTM!

**Pull request merged**